runtime/compiler/env/VMJ9.cpp

```diff
@@ -2488,7 +2488,7 @@ TR_J9VMBase::getClassSignature(TR_OpaqueClassBlock * clazz, TR_Memory * trMemory
    for (i = 0; i < numDims; i++)
       sig[i] = '[';
    if (* name != '[')
-      sig[i++] = 'L';
+      sig[i++] = 'L'; //same as above, do we need to worry about Q here?
```

Comment:
Same as previous comment - I think this will need to check whether myclass is an inline/value type class.